controllers/rhmi/rhmi_controller.go

```diff
@@ -832,7 +836,7 @@ func (r *RHMIReconciler) handleUninstall(installation *rhmiv1alpha1.RHMI, instal
 			if err != nil {
```

Comment:
can we have a look at above logic on 832 - if err > err != nil > !k8errIsNotFound looks a bit off.
If we are saying if err, I guess no point in checking if err != nil? WDYT? <not related to the PR but could be improved I think>